packages/typescript/tsconfig.json

```
@@ -10,7 +10,7 @@
         "allowJs": true
     },
     "files": [
-        "index.d.ts",
+        "types/index.d.ts",
```

nitpick: to catch any future + present type files

```suggestion
        "types",
```